CMakeLists.txt

```diff
@@ -109,6 +109,13 @@ if(WITH_XMF)
     include_directories(libwebm)
 
     add_subdirectory(libxmf)
+
+    add_executable(
```

Comment:
the sample C++ executable can be removed since you've got the C# test sample code that uses the C# bindings, no need to keep it

Reply:
updated